libraries/mbed/targets/hal/TARGET_RENESAS/TARGET_RZ_A1H/serial_api.c

```diff
@@ -512,7 +512,11 @@ int serial_getc(serial_t *obj) {
     int data;
     int was_masked;
 
+#if defined ( __ICCARM__ )
+    was_masked = __disable_irq_iar();
```

Comment:
https://github.com/mbedmicro/mbed/blob/master/libraries/mbed/targets/cmsis/core_cmFunc.h#L342. Or which one are you including here? I remember this void return type was an issue with this